OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -408,9 +409,10 @@ function Set-TargetResource {
         [int]$ServerPort = 10943,
         [string]$tentacleDownloadUrl = $defaultTentacleDownloadUrl,
         [string]$tentacleDownloadUrl64 = $defaultTentacleDownloadUrl64,
-        [ValidateSet("PublicIp", "FQDN", "ComputerName", "Custom")]
+        [ValidateSet("PublicIp", "FQDN", "ComputerName", "DHCP","Custom")]
         [string]$PublicHostNameConfiguration = "PublicIp",
         [string]$CustomPublicHostName,
+        [string]$CidrRange,
```

Comment:
May consider adding some input validation for CIDR range parameter here, so we won't get `IndexOutOfRangeException` when testing against each private IP addresses